.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/envoyproxy/go-control-plane
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/golang:1.11
```

Comment:
?? why

Reply:
I had 1.11 locally and thought the build might be failing due to some go-imports differences. I can remove this part.